src/config/bootstrap.ts

```diff
@@ -6,7 +6,7 @@ const config: ConfigInterface = {
   },
   modules: {
     latestVersion: '0.0.19',
-    oldestVersion: '0.0.16'
+    oldestVersion: '0.0.16',
```

Comment:
we should change the `.version` to add the trailing comma or call `yarn format` before committing

Reply:
👍